spec/requests/vendor_api/versioning_spec.rb

```diff
@@ -102,5 +102,32 @@
         expect(response.status).to eq(200)
       end
     end
+
+    context 'when specifying a prerelease version' do
+      before do
+        stub_const('VendorAPI::VERSIONS', { '1.1pre' => [VendorAPI::Changes::RetrieveApplications],
+                                            '1.2' => [VendorAPI::Changes::RetrieveSingleApplication] })
+      end
+
+      context 'and the environment is production' do
+        it 'the route is not processed' do
+          allow(HostingEnvironment).to receive(:production?).and_return(true)
+
+          get_api_request "/api/v1.2/applications?since=#{CGI.escape(1.day.ago.iso8601)}"
```

Comment:
Should this call be to v1.1, as that's the prerelease version we want to obscure in production?

Reply:
No. It will specifically exclude 1.1 version changes as they are configured in a pre-release version but will allow retrieval of version 1.2 as it's the latest available. I didn't want to assume that version changes were released in a specific order, even though this is an unlikely scenario.

Reply:
I see, so it can't use the bulk applications endpoint because that was in a prereleased version.

Reply:
Exactly.